crates/levm/src/primitives.rs

```diff
@@ -0,0 +1,6 @@
+pub use bytes::Bytes;
+pub use ethereum_types::{Address, H160, U256, U512};
+
+use ethereum_types::H256;
+
+pub type B256 = H256;
```

Comment:
What does this represent?

Reply:
> We used it in MLIR to represent a hash type with 32 bytes
ACK

> I can remove it cause it's not currently being used
I agree with this, better be added when its needed

Reply:
done!